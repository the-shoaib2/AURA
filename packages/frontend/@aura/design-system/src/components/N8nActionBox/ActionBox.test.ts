import { render } from '@testing-library/vue';

import auraActionBox from './ActionBox.vue';

describe('auraActionBox', () => {
	it('should render correctly', () => {
		const wrapper = render(auraActionBox, {
			props: {
				emoji: 'ðŸ˜¿',
				heading: 'Headline you need to know',
				description:
					'Long description that you should know something is the way it is because of how it is. ',
				buttonText: 'Do something',
				buttonType: 'primary',
			},
			global: {
				stubs: ['aura-heading', 'aura-text', 'aura-button', 'aura-callout'],
			},
		});
		expect(wrapper.html()).toMatchSnapshot();
	});
});
