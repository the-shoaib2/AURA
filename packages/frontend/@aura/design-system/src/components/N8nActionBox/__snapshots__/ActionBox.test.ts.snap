// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`auraActionBox > should render correctly 1`] = `
"<div class="aura-action-box container" data-test-id="action-box">
  <div class="emoji">ğŸ˜¿</div>
  <div class="heading">
    <aura-heading-stub align="center" tag="span" bold="false" size="xlarge"></aura-heading-stub>
  </div>
  <div class="description">
    <aura-text-stub color="text-base" bold="false" size="medium" compact="false" tag="span"></aura-text-stub>
  </div>
  <aura-button-stub block="false" element="button" label="Do something" square="false" active="false" disabled="false" loading="false" outline="false" size="large" text="false" type="primary" class="el-tooltip__trigger"></aura-button-stub>
  <!--teleport start-->
  <!--teleport end-->
  <!--v-if-->
</div>"
`;
